src/libsyntax/test.rs

```diff
@@ -52,7 +52,8 @@ struct Test {
     path: Vec<Ident> ,
     bench: bool,
     ignore: bool,
-    should_panic: ShouldPanic
+    should_panic: ShouldPanic,
+    serial: bool
```

Comment:
Please end the line with a comma, so there is less noise when new fields get added.